mechanics/TOC-REVIEW.schedule

```diff
@@ -15,7 +15,19 @@
 2021-05-13T08:30:00-07:00 | (Cancelled)
 2021-05-20T08:30:00-07:00 | Operations WG
 2021-05-27T08:30:00-07:00 | Eventing WG
-2021-06-03T08:30:00-07:00 | Docs WG
-2021-06-10T08:30:00-07:00 | Networking WG
+2021-06-03T08:30:00-07:00 | Networking WG
+2021-06-10T08:30:00-07:00 | DUX WG
 2021-06-17T08:30:00-07:00 | Security WG
-2021-06-24T08:30:00-07:00 | User Experience WG
+2021-06-24T08:30:00-07:00 | Productivity WG
+2021-07-01T08:30:00-07:00 | Client WG
+2021-07-08T08:30:00-07:00 | Sources WG
+2021-07-15T08:30:00-07:00 | Serving API WG
+2021-07-22T08:30:00-07:00 | Scaling WG
+2021-07-29T08:30:00-07:00 | Event Delivery WG
+2021-08-05T08:30:00-07:00 | no presentation
+2021-08-12T08:30:00-07:00 | (Cancelled)
```

Comment:
What's the difference between "no presentation" and "(Cancelled)"? (Is this a copy paste error from above?)

Reply:
Ugh, this is copypasta... thank you!